proposals/collection-literals.md

```diff
@@ -210,7 +210,7 @@ Through the use of the [`init`](#init-methods) modifier, existing APIs can direc
 ## Natural type
 [natural-type]: #natural-type
 
-In the absence of a *target type*, a non-empty literal can have a *natural type*.
+In the absence of a *constructible target type*, a non-empty literal can have a *natural type*.
```

Comment:
👍

Reply:
There doesn't appear to be an actual definition for `constructible` in the document. Consider rewording the first part of `Constructible Collection Types` to precisely define that term.

Reply:
Updated, thanks.